easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+versionsuffix = '-Python-%(pyver)s-PyTorch-1.4.0'
+
+homepage = 'https://github.com/pytorch/vision'
+description = " Datasets, Transforms and Models specific to Computer Vision"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = ['https://github.com/pytorch/vision/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['eb9afc93df3d174d975ee0914057a9522f5272310b4d56c150b955c287a4d74d']
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow-SIMD', '6.0.0'),
+    ('PyTorch', '1.4.0', '-Python-%(pyver)s'),
```

We have a rule for the official repository of easyconfigs to limit dependencies on a single toolchain to a single version of the package. Since `fosscuda/2019b` already has `torchvision-0.4.2-fosscuda-2019b-Python-3.7.4.eb` depending on PyTorch v1.3.1, this easyconfig should depend on the same version of PyTorch as it shares the same toolchain. This is not a problem though, as torchvision v0.5.0 is compatible with PyTorch v1.3.1.

```suggestion
    ('PyTorch', '1.3.1', versionsuffix),
```